addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -221,6 +222,7 @@ def _discuss_channel_member_format(self, fields=None, extra_fields=None):
                 "id": True,
                 "persona": {},
                 "seen_message_id": True,
+                "new_message_separator": True,
```

Comment:
We should not return the value of `new_message_separator` of other members, it should be a user specific value (-> `channel_info`?)